src/js/disability-benefits/components/Notification.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export default function Notification({ title, body, onClose, type }) {
```

Comment:
Looks like we have a title and that component doesn't. It's something to look into later, I think.